ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.12.yaml

```diff
@@ -0,0 +1,220 @@
+base_images:
+  rhel-8-golang-1.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- dockerfile_path: Dockerfile
+  from: rhel-8-golang-1.20
+  to: oadp-operator-1.3
+- dockerfile_path: build/ci-Dockerfile
+  from: rhel-8-golang-1.20
+  to: test-oadp-operator-1.3
+operator:
+  bundles:
+  - dockerfile_path: build/Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/konveyor/oadp-operator:oadp-1.3
+    with: oadp-operator
```

Comment:
I think the error may be in here: it should be the name of the image? `oadp-operator-1.3`?